02-data_uses.Rmd

```diff
@@ -73,6 +73,107 @@ Cohort identification is important regardless of research study type, but to pro
+
+Designing CRFs that are inclusive, accessible, and sensitive to the diverse needs of participants, particularly marginalized groups, requires careful consideration. Questions within CRFs should be formulated to gather comprehensive and accurate clinical data while ensuring participants feel safe, respected, and comfortable. This section explores the types of questions that can be asked using CRFs with a focus on **accessibility, sensitivity, inclusivity, and comfort for marginalized people**
+
+::: {.notice}
+
+While within a study, CRFs help to ensure standardization in data collection, there may be a lack of standardization when comparing data between studies if each study did not use the same CRFs or comparable/subsets of questions within forms. This section will provide guidelines for types of questions that may be found within CRFs and writing these questions; however, there aren't necessarily templates that have been used widely within the field. Further, your specific study needs may also require different or additional types of questions -- the guidelines within this section are not exhaustive/all-encompassing of what may be encountered within the field.
+
+:::
+
+Here, we explore the different categories of questions that may be included within case report forms:
+
+1. **Demographic and Socioeconomic Questions**
+
+Capturing demographic and socioeconomic data is fundamental in clinical research to understand the background and diversity of study participants. However, these questions must be asked in a manner that respects privacy, avoids assumptions, and is inclusive of all identities. It’s also good practice in order to respect privacy to include an option allowing the participants to leave the question unanswered or respond with “prefer not to answer”.
+
+- **Sex or Gender**: When asking about gender, it is crucial to go beyond binary options (e.g., male, female) and include a range of choices such as non-binary, genderqueer, or “prefer to self-describe.”
+- **Ethnicity and Race**: Ethnicity and race questions should use inclusive and respectful language, allowing participants to self-identify rather than selecting from a predefined list.
```

Comment:
```suggestion
- **Ethnicity and Race**: Ethnicity and race questions should be specific and use respectful language, allowing participants to self-identify rather than selecting from a predefined list.
```